### ISSUE TITLE
Display embedded social media feeds

### ISSUE THREAD

**Issue by texascloud** [CamelBackNotation/cs373-idb#15]
Someone should figure out how to do this using React. I'd suggest creating a Component that maybe takes something like a company name (ex: Ubisoft, Nintendo, etc), uses the Twitter API to perform a lookup for their account, then displays their timeline in whatever element that React Component is attached to. It may make sense to break this up into multiple Components if the logic becomes too involved. This will make the process and progress easier to test and also debug if anything doesn't work correctly. [This tutorial from Facebook shows how you can split your logic up into multiple Components](https://facebook.github.io/react/docs/tutorial.html)

Once that Component is made, we can simply reuse it anywhere we want to display an integrated Twitter feed! Try to first write unit tests that specifically test the created component (preferably before you implement it cuz ya know, TDD and all, but w/e :stuck_out_tongue_winking_eye: )

**TL;DR:**
- Create React Component to display Twitter feed
- Write tests to make sure Component works properly
- Re-use Component anywhere we need to display feed, hooray!!
- Report methods/tools used to achieve this to person working on technical report #16
